main.go

```diff
@@ -1177,20 +1170,27 @@ var importCmd = &cobra.Command{
 	RunE: func(cmd *cobra.Command, args []string) (err error) {
 		var in io.Reader
 		if len(args) > 0 {
-			var f *os.File
-			f, err = os.Open(args[0])
+			filename := args[0]
+			file, err := os.Open(filename)
```

Comment:
Can you revert your change here? I think it is not a good idea to use `something, err := doSomething()` inside a function with a named return `err error`. It can introduce some subtle bugs that will ignore some errors.